### PR TITLE
156 admin code fixes

### PR DESCRIPTION
**Pull request by bbean23**

### CODE REVIEW

File: opencsp/__init__.py [hunk 1]
```diff
@@ -66,8 +66,4 @@ def _opencsp_settings_dirs() -> list[str]:
 opencsp_settings = configparser.ConfigParser(allow_no_value=True)
 opencsp_settings.read(_settings_files)
 
-for section in opencsp_settings.sections():
```
Comment (e10harvey) [inline opencsp/__init__.py:69]: Is there a nightly example that causes this for-loop to fail?

File: contrib/scripts/sensitive_strings.py [hunk 2]
```diff
@@ -109,7 +109,9 @@ def _is_binary_file(self, file_path: str, file_name_ext: str):
         ext = ext.lower()
         if ext == ".ipynb":
             is_binary_file = True
-        if self._is_img_ext(ext):
+        elif ext == ".h5":
```
Comment (e10harvey) [inline contrib/scripts/sensitive_strings.py:112]: Is there no corresponding test update?

**Review by e10harvey — COMMENTED** [review]
The sensisit

File: contrib/scripts/sensitive_strings.py (hunk 2 above)
Comment (bbean23) [reply contrib/scripts/sensitive_strings.py]: Updated in commit https://github.com/sandialabs/OpenCSP/pull/157/commits/fd5c12d8c1cb7897c6429ec5c1338a3800c12577

**Review by braden6521 — APPROVED** [review]
Looks good to me

**Pull request merged**